skillful_network_client/src/app/home/news-feed/add-publication/add-publication.component.html

```diff
@@ -0,0 +1,17 @@
+
+<div class='card mat-elevation-z4' id="publication-card">
+    <mat-card>
+        <div class="div-principal">
+            <div>
+                <mat-card-title>
+                    <button mat-button (click)="openDialog()" id="title-publication"><span class="material-icons" id="create-icon">create</span> Créer un post</button>
```

Comment:
What do you think about putting directly a text input as on Facebook instead of the dialog? Like I can type directly my message from the newsfeed screen.

Reply:
Just checked Facebook, it is as you did. But what do you think about my idea?